command/action.go

```diff
@@ -30,8 +23,8 @@ func (c *ActionCommand) Help() string {
 	helpText := `
 Usage: nomad action [options] <action>
 
-  Perform a predefined command inside the environment of the given allocation
-  and task.
+  Perform a predefined command inside the environment of a given context.
+  Currently this acts as a wrapper around the job action command.
 
   When ACLs are enabled, this command requires a token with the 'alloc-exec',
```

Comment:
This paragraph is also specific to `nomad job action`. We can probably remove it.